pkg/validationfile/loader_test.go

```diff
@@ -15,39 +12,68 @@ import (
 
 func TestPopulateFromFiles(t *testing.T) {
 	tests := []struct {
-		name      string
-		filePaths []string
-		want      []*core.RelationTuple
+		name          string
+		filePaths     []string
+		want          []string
+		expectedError string
 	}{
 		{
 			name:      "no comment",
 			filePaths: []string{"testdata/loader_no_comment.yaml"},
-			want: []*core.RelationTuple{
-				tuple.Parse("example/project:pied_piper#owner@example/user:milburga"),
-				tuple.Parse("example/project:pied_piper#reader@example/user:tarben"),
-				tuple.Parse("example/project:pied_piper#writer@example/user:freyja"),
+			want: []string{
+				"example/project:pied_piper#owner@example/user:milburga",
+				"example/project:pied_piper#reader@example/user:tarben",
+				"example/project:pied_piper#writer@example/user:freyja",
 			},
+			expectedError: "",
 		},
 		{
 			name:      "with comment",
 			filePaths: []string{"testdata/loader_with_comment.yaml"},
-			want: []*core.RelationTuple{
-				tuple.Parse("example/project:pied_piper#owner@example/user:milburga"),
-				tuple.Parse("example/project:pied_piper#reader@example/user:tarben"),
-				tuple.Parse("example/project:pied_piper#writer@example/user:freyja"),
+			want: []string{
+				"example/project:pied_piper#owner@example/user:milburga",
+				"example/project:pied_piper#reader@example/user:tarben",
+				"example/project:pied_piper#writer@example/user:freyja",
 			},
+			expectedError: "",
 		},
 		{
 			name:      "multiple files",
 			filePaths: []string{"testdata/initial_schema_and_rels.yaml", "testdata/just_rels.yaml"},
-			want: []*core.RelationTuple{
-				tuple.Parse("example/project:pied_piper#owner@example/user:milburga"),
-				tuple.Parse("example/project:pied_piper#reader@example/user:tarben"),
-				tuple.Parse("example/project:pied_piper#writer@example/user:freyja"),
-				tuple.Parse("example/project:pied_piper#owner@example/user:fred"),
-				tuple.Parse("example/project:pied_piper#reader@example/user:tom"),
-				tuple.Parse("example/project:pied_piper#writer@example/user:sarah"),
+			want: []string{
+				"example/project:pied_piper#owner@example/user:milburga",
+				"example/project:pied_piper#reader@example/user:tarben",
+				"example/project:pied_piper#writer@example/user:freyja",
+				"example/project:pied_piper#owner@example/user:fred",
+				"example/project:pied_piper#reader@example/user:tom",
+				"example/project:pied_piper#writer@example/user:sarah",
 			},
+			expectedError: "",
+		},
+		{
+			name:      "multiple files",
+			filePaths: []string{"testdata/initial_schema_and_rels.yaml", "testdata/just_rels.yaml"},
```

Comment:
I think I mentioned this before, but what about when two have different schemas?

Reply:
They'd be combined